tests/integration/test_allclose.py

```diff
@@ -0,0 +1,284 @@
+# Copyright 2021-2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pytest
+
+import cunumeric as num
+
+SCALARS_TRUE_DEFAULT = (
+    (0, -1e-8),
+    (1e10, 1.00001e10),
+    (1 + 1j, 1 + 1.00001j),
+)
+
+SCALARS_TRUE_INF = (
+    (np.inf, np.inf),
+    (-np.inf, -np.inf),
+)
+
+
+@pytest.mark.parametrize(
+    "ab",
```

Comment:
I think you can do this instead:
```
"a", "b"
```
and then avoid the explicit tuple unpacking both in the test and in the `ids` function (this seems to be the case for several tests in this PR)

Reply:
Fixed.